.eslintrc

```diff
@@ -30,7 +30,7 @@
     "no-shadow": 2,
     "no-shadow-restricted-names": 2,
     "no-spaced-func": 2,
-    "no-trailing-spaces": 2,
+    "no-trailing-spaces": 0,
```

Comment:
It's not about file size, but rather about annoyance, especially when editor is configured to auto-trim them (e.g. via `.editorconfig`) and then you're getting weird diffs whenever you try to commit changes.